README.md

```diff
@@ -102,3 +102,4 @@ module.exports = schema;
 [Re:dash Dashboard]: https://sql.telemetry.mozilla.org/
 [Infernyx Homepage]: https://github.com/tspurway/infernyx
 [Joi Homepage]: https://github.com/hapijs/joi
```

Comment:
I'm not 💯  sure, but this `[Joi Homepage]` slug _may_ now be orphaned, since you changed to `[Joi-browser Homepage]` above.